recipe/meta.yaml

```diff
@@ -25,31 +25,31 @@ source:
     - no-cocoa-notification.patch  # [osx]
 
 build:
-  number: 3
+  number: 4
   skip: true  # [win]
   detect_binary_files_with_prefix: true
 
 requirements:
   build:
     - python
-    - xz
```

Comment:
That was probably due to the fact that the source is compressed with `xz`, but conda can take care of that and it is not a "real" dependency.